games-engines/gemrb/patches/gemrb-0.8.5.patchset

```diff
@@ -0,0 +1,47 @@
+From 91cc2f67a24e4648e151ec9034c7e8efc57f5b80 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Mon, 2 Apr 2018 19:08:54 +0200
+Subject: Haiku install paths
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index b5d2017..842a92c 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -63,6 +63,8 @@ ENDIF(PREFIX)
+ if (NOT LAYOUT)
+ 	if (WIN32)
+ 		set(LAYOUT "home")
++	elseif (HAIKU)
++		set(LAYOUT "haiku")
+ 	elseif (APPLE)
+ 		set(LAYOUT "bundle")
+ 		# favor mac frameworks over unix libraries
+@@ -111,6 +113,24 @@ elseif (${LAYOUT} MATCHES "fhs")
+ 	SET_PATH( SVG_DIR ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/scalable/apps )
+ 	SET_PATH( MENU_DIR ${CMAKE_INSTALL_PREFIX}/share/applications )
+ 	SET_PATH( EXAMPLE_CONF_DIR ${SYSCONF_DIR} )
++elseif (${LAYOUT} MATCHES "haiku")
++	SET_PATH( LIB_DIR ${CMAKE_INSTALL_PREFIX}/lib/gemrb )
```

Comment:
Would this work with secondaryArch?

Reply:
@extrowerk Please add detection for x86 secondary arch as well, according to the patch, the libs will ends up at `$prefix/lib` for secondary architecture, which may not be desirable.